subsys/zigbee/cli/zigbee_cli_cmd_zdo.c

```diff
@@ -1859,14 +1880,31 @@ static void print_bind_resp(const struct shell *shell,
 {
 	const zb_zdo_binding_table_record_t *tbl_rec;
 	uint32_t idx;
+	uint32_t group_bind_tbl_rec_len;
 	uint32_t next_start_index = ((uint32_t)bind_resp->start_index +
 				     bind_resp->binding_table_list_count);
 
+	/* Calculate bind_table record length used for groups binding:
+	 * - no dst_ep (1B)
+	 * - only short_addr (2B) instead of long_addr (8B)
+	 */
+	group_bind_tbl_rec_len = sizeof(*tbl_rec) - sizeof(tbl_rec->dst_endp) +
```

Comment:
Could be assuming that there will be no changes underneath it.
How would you like it to be defined?

Reply:
Am I correct that this is a structure, defined by the Zigbee specification?
From my perspective - this could be anything from sizeofs on type names to something like this:
```
/* In case of group binding, the addressing field contains a short address and the endpoint number is not included. */
ZB_OFFSETOF(zb_zdo_binding_table_record_t, dst_address) + sizeof(zb_uint16_t)
```

Reply:
I like it, will do that